omnibus/files/private-chef-cookbooks/private-chef/recipes/opscode-erchef.rb

```diff
@@ -59,6 +59,12 @@
   notifies :restart, 'runit_service[opscode-erchef]' unless backend_secondary?
 end
 
+template "/opt/opscode/embedded/service/opscode-erchef/vm.args" do
+  source "oc_erchef.vm.args.erb"
```

Comment:
I believe we'll eventually want to do this for the other erlang apps as well so we'll likely want this to be a more generic template that we can pass the app name into.